src/app/src/components/ReportFacilityStatus.jsx

```diff
@@ -128,7 +128,12 @@ function ReportFacilityStatus({
                         className={classes.description}
                     >
                         Please provide information the OAR team can use to
-                        verify this status change.
+                        verify this status change. Should you need to share or
+                        attach a document or screenshot, please email it to
```

Comment:
We need to force a " " in here between "to" and the email address. <img width="845" alt="Screen Shot 2021-08-06 at 9 00 30 AM" src="https://user-images.githubusercontent.com/17363/128539306-66bea217-25a4-4a77-9a06-9685a167ae12.png">

Reply:
Updated! Good catch 👍